config/labels/analyzers/clang-tidy.json

```diff
@@ -5078,6 +5078,13 @@
       "profile:sensitive",
       "severity:LOW"
     ],
+    "misc-confusable-identifiers": [
+      "doc_url:https://clang.llvm.org/extra//clang-tidy/checks/misc/confusable-identifiers.html",
+      "profile:default",
+      "profile:extreme",
+      "profile:sensitive",
+      "severity:LOW"
```

Comment:
Should something that detects a CVE be of *LOW* severity (and not included in the security profile)?

Reply:
It has a CVE but it can be argued about the risk....
I don't mind changing it if you prefer :)
I don't have strong opinion about it

Reply:
Meh, even the CVE site linked in the Tidy docs starts with `DISPUTED`. It's a strange attack vector from the get-go, e.g., for me the following two lines render **completely** differently:

```cpp
int fo;
int 𝐟o;
```

![Visual render example](https://user-images.githubusercontent.com/1969470/182846044-ff2fdf9e-6937-4255-a89f-3c79db4f1a30.png)

Think this will be okay as is.

Reply:
If you are curious to know more :
https://opensourcesecuritypodcast.libsyn.com/episode-296-is-trojan-source-a-vulnerability
;)